modules/common/src/main/mon.scala

```diff
@@ -159,8 +159,13 @@ object mon {
       object trace {
         def create = makeTrace("round.move.trace")
       }
-      val networkLag = rec("round.move.network_lag")
-      val lagLowEstimate = rec("round.move.lag_low_estimate")
+      val networkLag = rec("round.move.network_lag") // Deprecated.
+      val avgClientLag = rec("round.move.avg_client_lag")
+      val lagEstimateError = rec("round.move.lag_estimate_error")
+      val lagCompDeviation = rec("round.move.lag_comp_deviation")
+      def uncompedLag(key: String) = rec(s"round.move.uncomped_lag.$key")
```

Comment:
what does `key` represent here? It could be named better maybe

Reply:
wasn't sure of a good name, sort of abstract where bigger number = higher quota.

It's tenths of a second of the lag comp you get per move.